CHANGELOG.md

```diff
@@ -2,13 +2,37 @@
 
 # Master
 
-- [ENHANCEMENT] Add option `--insecure-proxy` to `serve` to allow for proxying
-  self signed SSL certificates.
-
 ### Applications
 ### Addons
 ### Blueprints
 
+### 0.1.7
+
+- Users
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
+  + Replace `ember-cli-esnext` with `ember-cli-6to5`:
```

Comment:
This could be changed to something like:

Uninstall `ember-cli-esnext`. It has been replaced by `ember-cli-6to5` which should be added to your project from the upgrade steps.